concent_api/core/tests/test_payments_sci_callback.py

```diff
@@ -0,0 +1,192 @@
+from core.payments.sci_callback import sci_callback
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
+(DIFFERENT_CONCENT_PRIVATE_KEY, DIFFERENT_CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
+(SIGNING_SERVICE_PRIVATE_KEY, SIGNING_SERVICE_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY=CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY=CONCENT_PUBLIC_KEY,
+    SIGNING_SERVICE_PUBLIC_KEY=SIGNING_SERVICE_PUBLIC_KEY,
+)
+class DatabaseTransactionsStorageTest(TestCase):
+
+    multi_db = True
+
+    def setUp(self):
+        super().setUp()
+
+        self.v = 28
+        self.r = 47275086640460304014058339281073197467190165829103506190445899072480758914192
+        self.s = 23780459904358005364906574578927371041875210913701178203010728904969030802094
+
+        self.frame = append_frame_separator(
+            escape_encode_raw_message(
+                GolemMessageFrame(
+                    payload=self._create_signed_transaction(),
+                    request_id=99
+                ).serialize(
+                    SIGNING_SERVICE_PRIVATE_KEY
+                )
+            )
+        )
+
+    def _create_transaction_signing_request(self):  # pylint: disable=no-self-use
+        transaction_siging_request = TransactionSigningRequest(
+            nonce=99,
+            gasprice=10 ** 6,
+            startgas=80000,
+            value=10,
+            to='7917bc33eea648809c28',
+            data=b'3078666333323333396130303030303030303030303030303030303030303030303032393662363963383738613734393865663463343531363436353231336564663834666334623330303030303030303030303030303030303030303030303030333431336437363161356332633362656130373531373064333239363566636161386261303533633030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303237313030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303562343336643666',
+        )
+        setattr(transaction_siging_request, 'from', '7917bc33eea648809c29')
+        return transaction_siging_request
+
+    def _create_signed_transaction(self):
+        return SignedTransaction(
+            nonce=99,
+            gasprice=10 ** 6,
+            startgas=80000,
+            value=10,
+            to=b'7917bc33eea648809c28',
+            v=self.v,
+            r=self.r,
+            s=self.s,
+            data=b'3078666333323333396130303030303030303030303030303030303030303030303032393662363963383738613734393865663463343531363436353231336564663834666334623330303030303030303030303030303030303030303030303030333431336437363161356332633362656130373531373064333239363566636161386261303533633030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303237313030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303562343336643666',
```

Comment:
If this is not some specific byte string for this transaction I think it might be better to generate this from `os` lib by function `urandom`